completion.sh

```diff
@@ -0,0 +1,325 @@
+# bash completion V2 for gh                                   -*- shell-script -*-
```

Comment:
It's probably worth pointing out: This is an auto-generated file. It's created when `tools/prep_release` runs in order to provide bash completion in the snap

Reply:
should really generate within a workflow instead (rather than hard-code here).

Reply:
The `prep_release.sh` file gets run as a callback by the `sync-gh-releases` workflow in `upstream-release`. I've deleted this file though so it requires generating the completion for the snap. (Documentation will be included :-) )